src/main/java/com/karimbouchareb/chromatographysimulator/ui/SplashScreen.java

```diff
@@ -24,52 +24,55 @@
 
 public class SplashScreen {
     private StackPane splashScreenPane = new StackPane();
-    private ImageView title;
-    private Pane titlePane;
-    private Pane baselineAnimationPane;
-    private Pane travisHeadPane;
-    private ImageView backgroundScatter;
-    private Pane launchPane;
-    private ImageView travisHead;
 
     /**
      * This is called when {@link ChromatographySimulatorApp#init()} is called. It inits each of the ui components of
      * the splash screen: {@link #initBackground()}, {@link #initTravisHead()}, {@link #initTitle()},
-     * {@link #initBaselineAnimation()}, and {@link #initLaunchButton()}.
+     * {@link #initBaselineAnimation(double, double)}, and {@link #initLaunchButton(double, ImageView)}
      */
     public void initSplashScreen(){
         splashScreenPane.setBackground(Background.fill(Color.BLACK));
-        initBackground();
-        initTravisHead();
-        initTitle();
+        // Background Scatter
+        ImageView backgroundScatter = initBackground();
+        // Travis Head
+        Pane travisHeadPane = new Pane();
+        ImageView travisHead = initTravisHead();
+        travisHeadPane.getChildren().add(travisHead);
```

Comment:
Instead of initializing travisHeadPane first, then creating the child travisHead, then adding, reverse the order so you can pass the child in:
```
ImageView travisHead = initTravisHead();
Pane travisHeadPane = new Pane(travisHead);
```